### DISCUSSION TITLE
Use fonts embedded via base64 url

### DISCUSSION THREAD

**Discussion by moritzgloeckl** [mpdf/mpdf#1710] [General]
Hi,

is it possible to embed ttf fonts in the HTML without having to call any PHP methods or use ttf files on disk? I'm thinking of embedding TTF fonts via base64 urls or similar. Users could then take advantage of managing their own HTML templates without dealing with ttf on the filesystem.

Thanks!

**Comment by finwe** [comment]
No, it is not.
